src/components/PasswordField/PasswordField.tsx

```diff
@@ -1,24 +1,26 @@
 import React, { useState } from 'react';
```

Comment:
You don't have to import `React` in React 18 (our version). You should remove this and just keep the `{ useState }` part.